client/src/components/Editor/DraftEditor.js

```diff
@@ -18,31 +18,8 @@ import { makeStyles } from "@material-ui/core/styles";
 import Toolbar from "./Toolbar";
 
 // Prism language support:
-import {
-  go,
-  clike,
-  ruby,
-  php,
-  python,
-  c,
-  cpp,
-  csharp,
-  java,
-  languagesGrammar,
-} from "./languages";
```

Comment:
I don't think that without languages we will have prism highlighting. Have u tested it?

Reply:
yes prism editor still has highlighting after I remove the languages import. None of the languages were used in the file, thus the reason why I removed it.